build-configs/BuildConfigType.js

```diff
@@ -30,6 +30,7 @@ export type FeatureFlagsType = {|
 // Available on all platforms
 export type CommonBuildConfigType = {|
   appName: string,
+  appDescription: string,
```

Comment:
Maybe we should move this to web. I think we don't want to use it for native. I think we should also add a comment why and where this is used.

Reply:
done